site/cypress/e2e/Cypress-Gradeable/cyclic_grading_view.spec.js

```diff
@@ -0,0 +1,53 @@
+describe('Cyclic grading View Test', () => {
+    ['ta', 'instructor'].forEach((user) => {
+        it(`${user} cyclic grading view testing for Grading Homework`, () => {
+            cy.login(user);
+            cy.visit(['sample', 'gradeable', 'grading_homework', 'grading', 'status']);
+            cy.get('[data-testid="Grading Index"]').should('exist').click();
+            if (user === 'ta') {
+                cy.get('[data-testid="popup-window"]').should('exist');
+                cy.get('[data-testid="close-button"]').should('exist');
+                cy.get('[data-testid="close-hidden-button"]').should('exist');
+                cy.get('[data-testid="agree-popup-btn"]').should('exist').click();
+            }
+            cy.get('[data-testid="view-sections"]').click();
+            cy.get('[data-testid="grade-button"]').eq(12).click();
+            cy.get('[data-testid="student-info-btn"]').click();
+            cy.get('.rubric-title').should('contain', 'Joe Student');
```

Comment:
Can you make this more specific, and add a test-id for the element you are trying to test? 
Apply for each instance of this